src/assets/scss/featuredarts.scss

```diff
@@ -0,0 +1,162 @@
+#featuredarts {
+  width: 100%;
+
+}
+
+#feature_list{
+  padding:5%;
+  background: #E5E5E5 !important;
+}
+
+#featured_arts_header h5 {
+  font-family: 'Fredericka the Great', cursive;
+  font-style: normal;
+  font-weight: normal;
+  line-height: normal;
+  font-size: 30px;
```

Comment:
Please try to use `em` as often as possible. It has great advantage over `px`

Reply:
Okay will do

Reply:
You haven't changed this.

Reply:
And every other one.